packages/p2p/src/components/my-profile/my-profile.scss

```diff
@@ -1,10 +1,21 @@
 .my-profile {
+    display: flex;
+
+    @include mobile {
+        display: block;
+    }
+
     &__content {
+        display: flex;
+        flex-direction: column;
+        flex: 1;
```

Comment:
Need to add `flex: 1` to ensure my profile contents are filled to the viewport width:
### Before
![Screenshot 2022-08-03 at 12 28 07 PM](https://user-images.githubusercontent.com/103016120/182527633-93d19793-ee51-4f9c-9b6e-82e44e169289.png)


### After
![Screenshot 2022-08-03 at 12 30 31 PM](https://user-images.githubusercontent.com/103016120/182527662-d09f80c9-aa0c-4672-8683-141190476cd9.png)